### DISCUSSION TITLE
Is a version for Android TV possible?

### DISCUSSION THREAD

**Discussion by Neglegunt** [aniyomiorg/aniyomi#1245] [Ideas]
what the title says.

I tried using the app on an Android TV, but the UX is designed for touch versions, so there comes a point where moving around with the TV remote is impossible due to such a limitation.

Beyond that inconvenience, in my smartphone works perfectly. I love this app!

**Comment by hiteshFoss** [comment]
@Neglegunt 
I'm using the preview version (App version: 0.14.6-6691) and fireTV; and I think the experience is quite managable to navigate using TV remote for _anime only_.

Else you can always use a virtual mouse control. 

Again, if we are considering manga in scenario, then we shouldn't consider it

**Reply by LittleFreak** [reply]
Unfortunately, amazon will deprecate from Android on their fire TV devices, so there goes one reason for porting into the void 🕳️

**Comment by LittleFreak** [comment]
See #162

**Comment by Shadowstepcomet** [comment]
You can install it but remote is difficult here is a solution that works fine download Bluetooth keyboard and mouse by appground io this uses your phone as mouse to use on your android TV box via Bluetooth it might be slower than remote but it works.
